libcontainer/factory_linux.go

```diff
@@ -224,10 +247,15 @@ func (l *LinuxFactory) Load(id string) (Container, error) {
 		config:               &state.Config,
 		initArgs:             l.InitArgs,
 		criuPath:             l.CriuPath,
-		cgroupManager:        l.NewCgroupsManager(state.Config.Cgroups, state.CgroupPaths),
 		root:                 containerRoot,
 		created:              state.Created,
 	}
+	resourceManagers := make(map[string]resourcemanager.ResourceManager)
+	resourceManagers["cgroups"] = l.NewCgroupsManager(state.Config.Cgroups, state.CgroupPaths)
+	if intelrdt.IsIntelRdtEnabled() {
+		resourceManagers["intelrdt"] = l.NewIntelRdtManager(&state.Config, id)
```

Comment:
`state.IntelRdtPath` is ignored when load container from disk. This will result in residual directory in `/sys/fs/resctrl`.

Steps to reproduce:
```
# runc create testcontainer
# ls /sys/fs/resctrl
cpus  info  schemata  tasks  testcontainer
# runc delete testcontainer
# ls /sys/fs/resctrl
cpus  info  schemata  tasks  testcontainer
```

Seems that `NewIntelRdtManager` needs one more parameter to record `IntelRdtPath`.